CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 3.5.0)
+cmake_minimum_required(VERSION 3.31.0)
```

Comment:
We aren't using any of the super modern CMake futures so version 3.5.0 is good enough.